plugins/org.jboss.tools.openshift.common.core/src/org/jboss/tools/openshift/internal/common/core/util/CommandLocationLookupStrategy.java

```diff
@@ -29,12 +29,22 @@
 	private static final String LINUX_PATHVAR = "PATH";
 	private static final String WINDOWS_PATHVAR = "Path";
 
+	private static final String LINUX_SEPARATOR = ":";
```

Comment:
This would be wrong, because then LINUX_STRATEGY would have incorrect values if instantiated while on a windows machine.  I admit this might not be a good usecase, but we never know how people may use these classes in the future, and it's important the LinuxStrategy is instantiated with the correct separator even when on windows.